.github/workflows/tests.yaml

```diff
@@ -41,5 +41,7 @@ jobs:
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
           files: ${{ github.workspace }}/coverage.xml
-          fail_ci_if_error: true
+          # Don't fail build if coverage can't be uploaded because the
+          # CODECOV_TOKEN is not shared with PRs from forks.
+          fail_ci_if_error: false
```

Comment:
We can't get Codecov reports on PRs from forks unless we share the CODECOV_TOKEN.

Reply:
Or maybe not? https://github.com/kmax12/gridstatus/pull/322#issuecomment-1904572360

Reply:
The token is not required for public repos. https://docs.codecov.com/docs/frequently-asked-questions

> The upload token is required for all uploads, except originating from public projects using Travis-CI, Circle CI, Azure, Github Actions.